Dockerfile

```diff
@@ -114,20 +114,18 @@ ENV VERGEN_GIT_DESCRIBE=${VERGEN_GIT_DESCRIBE}
 
 # Network access: cargo auditable needs it
 RUN --network=default \
+  --mount=type=cache,target=/root/.cargo/registry \
+  --mount=type=cache,target=/app/target \
```

Comment:
I can't easily tell, but should these have `id=mas-<unique ID>` so that they don't share the same cache with other dockerfiles that have caches in these locations?
(Under podman it seems like they might just hash the `target` option and use that as the cache key)

Reply:
Erf, not sure it really matters honestly? 🤷 
We can always change that later